.github/workflows/rpm-build-and-test.yml

```diff
@@ -411,70 +224,95 @@ jobs:
       CP_EL8_VERSION:
       CP_EL9_VERSION:
       CP_LEAP15_VERSION:
+      CP_RPM_TEST_VERSION:
       DISTRO:
       CLUSTER_REQUEST_reqid:
       STAGE_NAME:
       QUEUE_URL:
       LABEL:
+      STAGE_TAGS:
+      FTEST_ARG:
       DISTRO_NAME:
       DISTRO_VERSION:
       COMMIT_STATUS_DISTRO_VERSION:
     steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+          fetch-depth: 500
+          ref: ${{ github.event.pull_request.head.sha }}
       - name: Import commit pragmas
-        uses: ./.github/actions/import-commit-pragmas
+        uses: daos-stack/action-import-commit-pragmas@v1
+        with:
+          commit-message: ${{ needs.Call-RPM-Build.outputs.dequoted-commit-message }}
       - name: Set variables
         run: |
             set -eux
             env
-            STAGE_TAGS="-hw"
+            STAGE_TAGS="vm"
             FTEST_ARG=""
-            INST_RPMS="daos-client daos-tests daos-server daos-serialize daos-tests-internal"
-            case "${{ matrix.distro }}" in
+            TARGET=${{ github.event_name == 'pull_request' &&
+                       github.event.pull_request.base.ref || github.ref }}
+            echo ${{ format('{0}', github.event.pull_request.base.ref) }}
+            echo "NEXT_VERSION == \"${{ env.NEXT_VERSION_[format('{0}',
+                                                         github.event.pull_request.base.ref)] }}\""
+            TARGET="${GITHUB_HEAD_REF:-$GITHUB_REF_NAME}"
+            INST_RPMS=daos-{server,client,tests{,-internal},serialize}
+            # TODO: replace 'master' below with the actual [target, in the case of PRs] branch name
```

Comment:
is this TODO comment still needed?

Reply:
Doesn't seem like it.  I can fix if another push is needed.